app/controllers/concerns/content_navigation_ab_testable.rb

```diff
@@ -0,0 +1,86 @@
+module ContentNavigationABTestable
+  CONTENT_NAVIGATION_TEST_NAME = "ContentNavigation".freeze
+  CONTENT_NAVIGATION_DIMENSION = 67
```

Comment:
You should freeze this one too

Reply:
I think this confuses constants and frozen objects. 

Constants aren't actually constants in Ruby, and you can change them (albeit with the warning you're seeing).

Unrelatedly objects in Ruby can also be frozen, which means you can't mutate them.

```